### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kateryna3k**
don't judge harshly, I made a task with dotenv's help because in another way it doesn't work

### CODE REVIEW

**Review by Gr1zzZzzly — APPROVED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        URL,
+        params={"key": API_KEY, "q": CITY},
+    )
+    print(response.json())
```
Comment (TPolina) [inline app/main.py]: delete unnecessary print

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+    city = response.json()["location"]["name"]
+    country = response.json()["location"]["country"]
+    temp = response.json()["current"]["temp_c"]
+    date = response.json()["location"]["localtime"]
+    condition = response.json()["current"]["condition"]["text"]
```
Comment (TPolina) [inline app/main.py]: why do you repeat `json()` each time?

**Review by TPolina — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (kateryna3k) [reply app/main.py]: my bad( my program did't work and i tried different ways

**Review by TPolina — APPROVED** [review]
Well done!

**Pull request closed without merging**